src/class.js

```diff
@@ -5,10 +5,17 @@
 // Add a method called `comparePasswords`.  `comparePasswords` should have a parameter
 // for a potential password that will be compared to the `password` property.
 // Return true if the potential password matches the `password` property.  Otherwise return false.
-
-
-/* eslint-disable no-undef */ // Remove this comment once you write your classes.
-
+class User {
+  constructor(options) {
+    this.email = options.email;
+    this.password = options.password;
+  }
+  comparePasswords(pwdString) {
+    if (pwdString === this.password) return true;
```

Comment:
Just do `return pwdString === this.password;` . It automatically converts it into a bool